lib/resources/r4/related_person.yaml

```diff
@@ -131,25 +130,25 @@ fields:
               "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/period"
             },
             {
-              'valueCodeableConcept': {
-                'coding': [
+              "valueCodeableConcept": {
+                "coding": [
                   {
-                    'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40',
-                    'code': '155'
+                    "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+                    "code": "SIS"
                   }
                 ]
               },
-              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/relation'
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/relation"
             }
           ]
         }
       ]
     }
   note: |
     <ul>
-      <li> The <code>relationship.coding</code> must contain exactly one proprietary system. </li>
-      <li> The <code>relationship.coding</code> cannot contain a standard system. </li>
       <li> When specifying a `period`, the fields must include a time component with a timezone. </li>
+      <li>Each `coding` can be either from the standard or proprietary system, but not both.</li>
+      <li>If the coding is from a proprietary system, only one code may be provided.</li>
```

Comment:
nit: Add spaces after \<li> and before \</li>

Reply:
Changed throughout the file https://github.com/cerner/fhir.cerner.com/pull/662/commits/300867b2eb8e04f2456f4a36f5c45cc93172f23b#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR110-R113